packages/esm-commons-lib/src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -145,6 +146,7 @@ export const EncounterList: React.FC<EncounterListProps> = ({
               encounter.uuid,
               null,
               workspaceWindowSize,
+              patientUuid,
```

Comment:
Am still skeptical about breaking changes. Don't we have an instance of the patient in the workspace? Not sure we need this.

Reply:
https://github.com/openmrs/openmrs-esm-patient-chart/blob/6d0b682a14a2265d38d3544a9d54f0e5672d9ae7/packages/esm-patient-common-lib/src/form-entry-interop.ts#L55